### DISCUSSION TITLE
lien dans un tooltip

### DISCUSSION THREAD

**Discussion by florentarneodo** [MTESSDev/FRW#121] [Annonces]
Bonjour,
Je fais un tooltip, je peux utiliser des br, i dedans, mais pas un a qui fait bugger tout simplement le formulaire arrivé à cet endroit.
Est-ce normal? les liens a sont interdits ou vous pouvez les rajouter?
Merci et bonne journée.

**Comment by j0ber** [comment]
Bonjour,
S.v.p. nous fournir votre yaml de configuration de ce champ afin que nous puissions analyser.
Merci!

**Comment by florentarneodo** [comment]
[RSP-079-001-AIG.form.zip](https://github.com/MTESSDev/FRW/files/12194837/RSP-079-001-AIG.form.zip)

voir ligne 214 :)  enlever les commentaires

Merci et bonne journée

**Comment by j0ber** [comment]
Le problème est les doubles quotes qui viennent avec le lien. Elles brisent le html. Normalement le tooltip devrait afficher des textes relativement courts et simples.

Cela dit, dans votre situation il suffit d'encoder vos " (les remplacer par &quote;). Comme ceci  :  

`<a href=&quot;https://www.retraitequebec.gouv.qc.ca/fr/publications/nos-programmes/rrsp/participants/rachat-de-service/Pages/rachats-de-service.aspx&quot; target=&quot;blank&quot;><i>Les rachats de service</i></a>`

Le tooltip va alors fonctionner avec votre lien.

Nous allons créer un issue sur le sujet afin que ce soit géré automatiquement.

Merci de nous avoir signalé cette situation!

**Reply by florentarneodo** [reply]
Merci!
Bonne fin de semaine
